src/log4net/Util/SystemInfo.cs

```diff
@@ -514,7 +452,7 @@ public static string AssemblyLocationInfo(Assembly myAssembly)
     /// </remarks>
     public static string AssemblyQualifiedName(Type type)
     {
-      return type.FullName + ", " + type.Assembly.FullName;
+      return $"{type.FullName}, {type.Assembly.FullName}";
```

Comment:
Can you delete this method and replace the only call site with type.AssemblyQualifiedName?
(also cf legacy)